progress/internal/linear-progress.ts

```diff
@@ -33,9 +34,10 @@ export class LinearProgress extends Progress {
         (this.indeterminate ? 1 : this.buffer / this.max) * 100
       }%)`,
     };
-
+    // only display dots when visible - this prevents invisible infinite animation
+    const showDots = !this.indeterminate && (this.buffer >= this.max || this.value >= this.max) ;
     return html`
-      <div class="dots"></div>
+      ${when(showDots, () => html`<div class="dots"></div>`)}
```

Comment:
Can we swap this to a ternary with `nothing` for consistency? We don't use the `when()` directive elsewhere.

```ts
${showDots ? html`<div class="dots"></div>` : nothing}
```

Reply:
Sure - was wondering the same when writing it.